.env.sample

```diff
@@ -1,14 +1,24 @@
-# Testnet RPC URL
-SEPOLIA_RPC_URL=https://rpc.sepolia.org
+# Deployer private key
+DEPLOYER_PRIVATE_KEY=0x0000000000000000000000000000000000000000000000000000000000000001
 
+# Testnet RPC URL
+SEPOLIA_RPC_URL=https://rpc.sepolia.dev
 # Mainnet RPC URL
 MAINNET_RPC_URL=https://rpc.ankr.com/eth
 
-# Deployer private key
-DEPLOYER_PRIVATE_KEY=0x0000000000000000000000000000000000000000000000000000000000000001
-
 # Contract Verification
 ETHERSCAN_API_KEY=
-
 # Local Node
-ANVIL_PRIVATE_KEY=
\ No newline at end of file
+ANVIL_PRIVATE_KEY=
+
+REPORT_GAS=true
+
+MNEMONIC="test test test test test test test test test test test junk"
+
+OPTIMISTIC_ETHERSCAN_API_KEY=
```

Comment:
We're going to need this to scale up to potentially many networks (like 1-4 at launch, then potentially 10 within the year). Can you add comments here that organize the environment variables by network? (i.e. put all the PGN stuff toegether, Fantom stuff together, etc etc)